code/22-decay-tree-fitter/ntuple_DTF1.py

```diff
@@ -0,0 +1,39 @@
+from GaudiConf import IOHelper
+from Configurables import DaVinci, DecayTreeTuple
+from DecayTreeTuple.Configuration import *
+
+# Stream and stripping line we want to use
+stream = 'AllStreams'
+line = 'D2hhCompleteEventPromptDst2D2RSLine'
+
+# Create an ntuple to capture D*+ decays from the StrippingLine line
+dtt = DecayTreeTuple('TupleDstToD0pi_D0ToKpi')
+dtt.Inputs = ['/Event/{0}/Phys/{1}/Particles'.format(stream, line)]
+dtt.Decay = '[D*(2010)+ -> ^(D0 -> ^K- ^pi+) ^pi+]CC'
+
+# add a kinematic fitter
+from Configurables import TupleToolDecayTreeFitter,TupleToolDecay
```

Comment:
Using the [stuff above](https://github.com/lhcb/first-analysis-steps/pull/68/files#r31299573) you don't need these imports, but just a note that keeping all imports at the top of the file is [recommended style](https://www.python.org/dev/peps/pep-0008/#imports).